src/main/features/response/views/eligibility/how-much-paid-claimant.njk

```diff
@@ -0,0 +1,44 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, radioGroup, saveAndContinueButton, errorSummary %}
+{% from "table.njk" import tableStart, tableEnd, row, rowWithMultipleValue %}
+
+{% set title = t('Money claim') %}
+
+{% set heading = t('How much have you paid the claimant?') %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+
+      <form method="post" class="analytics-submit-event-trigger" data-event-action="Response: Select full admission option" data-event-label-from="option" novalidate>
```

Comment:
Having `Response: Select full admission option` is not readable night for me. Remember when somebody finds it in GA it should be obvious what it is. Please give it more meaningful name.